README.md

```diff
@@ -83,6 +84,42 @@ GLHF! <3
 
 GLHF! <3
 
+## Tantangan
+
+Anda mempunyai satu bulan penuh pada bulan November untuk membuat sebuah permainan *bebas* dengan bertemakan **peretasan, modifikasi dan/atau penggandaan**.
+
+Apa yang kita maksud dengan **bebas** berdasarkan pada peretasan, modifikasi dan/atau penggandaan? Kami sebenarnya bermaksud, *bebas* yang disarankan. Beberapa contoh mungkin seperti ini:
+
+* seorang pelari yang tak berujung dimana anda *retas* pohon biner menurun di jalur anda dengan kapak yang berpiksel,
+* sebuah pengambilan secara *modern* pada sesuatu yang klasik misalnya sebuah roguelike yang diatur di dalam dunia 3D atau dunia VR, atau
+* sebuah *penggandaan* permainan yang nyata membawa gabungan gurita/kucing kedalam dunia nyata.
+
+Tunjukan kreatifitasmu. anda dapat bekerja sendiri atau dengan tim dan membangun untuk berbagai platform ataupun perangkat. Penggunaan mesin game dari sumber terbuka dan berbagai pustaka itu dianjurkan tetapi tidak diperlukan.
+
+Kita akan tetap memantau beberapa blog GitHub favorit kami, dan dunia dapat menikmati (dan bahkan mungkin berkontribusi dan/atau untuk belajar) dengan kreasi anda.
+
+## Bagaimana cara berpartisipasi
+
+* [Daftarkan untuk akun pribadi gratis][github-signup] jika anda belum mempunyainya.
```

Comment:
"_Daftarkan akun_ ... jika Anda belum punya."

Please refer to the official Indonesian's glossary of IT-related terms.